src/pages/Encounters/EncounterList.tsx

```diff
@@ -741,17 +741,19 @@ export function EncounterList({
                           {t(`encounter_priority__${encounter.priority}`)}
                         </Badge>
                       </div>
-                      <Separator className="my-2" />
-                      <Link
-                        href={`/facility/${facilityId}/patient/${encounter.patient.id}/encounter/${encounter.id}/updates`}
-                        className="text-sm text-primary hover:underline text-right flex items-center justify-end group-hover:translate-x-1 transition-transform"
-                      >
-                        View Details
-                        <CareIcon
-                          icon="l-arrow-right"
-                          className="ml-1 h-4 w-4"
-                        />
-                      </Link>
+                      <div className="">
+                        <Separator className="my-2" />
+                        <Link
+                          href={`/facility/${facilityId}/patient/${encounter.patient.id}/encounter/${encounter.id}/updates`}
+                          className="text-sm text-primary hover:underline text-right flex items-center justify-end group-hover:translate-x-1 transition-transform"
+                        >
+                          View Details
```

Comment:
to checkout the readme, and kindly follow the merge checklist; translations are missing